src/renderer/components/ft-icon-button/ft-icon-button.js

```diff
@@ -60,25 +60,48 @@ export default Vue.extend({
   },
   methods: {
     toggleDropdown: function () {
-      $(`#${this.id}`)[0].style.display = 'inline'
-      $(`#${this.id}`).focus()
+      const $el = $(`#${this.id}`)
+      const el = $el[0]
 
-      $(`#${this.id}`).focusout(() => {
-        const shareLinks = $(`#${this.id}`).find('.shareLinks')
+      if (this.dropdownShown) {
+        el.style.display = 'none'
+        this.dropdownShown = false
+      } else {
+        el.style.display = 'inline'
+        $el.focus()
```

Comment:
To the best of my knowledge, JQuery .focus() calls are deprecated and should be replaced with vanilla JS .focus() calls.